conda/recipes/ucx-py/meta.yaml

```diff
@@ -43,14 +36,7 @@ requirements:
     {% for r in data.get("project", {}).get("dependencies", []) %}
     - {{ r }}
     {% endfor %}
-    # Something seems wrong with the run export as well now
-    {% if cuda_major == "11" %}
     - ucx
```

Comment:
Just wanted to check that this `ucx` is required in the `run` section. I noticed it was added by https://github.com/rapidsai/ucx-py/pull/986/files#diff-516b294659252e02809cd6547b97b86e8172251ad27a8cd5a725261e36e18d7c, but it was not removed here when reverting those changes. Is that intentional @vyasr ?

Reply:
Ah no good catch, that is not intentional. It was there because there were issues with the run exports on the C++ package. I'll open a PR to remove it now and we can verify that everything works correctly.

Reply:
Opened https://github.com/rapidsai/ucx-py/pull/990